src/Console/Command/GenerateCommand.php

```diff
@@ -96,25 +100,76 @@ protected function execute(InputInterface $input, OutputInterface $output)
         try {
             $this->state->setAreaCode(\Magento\Framework\App\Area::AREA_ADMINHTML);
 
-            $this->cacheManager->flush($this->cacheManager->getAvailableTypes());
+            // TODO: decide whether cache flushing is really required. temporally commented.
+            //$this->cacheManager->flush($this->cacheManager->getAvailableTypes());
```

Comment:
I'm pretty sure we only need to flush the full page cache. It would be much better